src/imu/imu.c

```diff
@@ -71,17 +58,24 @@ void imu_calibrate_zero(){
 		int zeroValue = g_imu_filter[physicalChannel].current_value;
 		//adjust for gravity
 		float countsPerUnit = imu_device_counts_per_unit(physicalChannel);
-		if (c->physicalChannel == IMU_CHANNEL_Z) zeroValue-= (countsPerUnit * (c->mode != MODE_IMU_INVERTED ? 1 : -1));
+		if (i == IMU_CHANNEL_Z) zeroValue -= (countsPerUnit * (c->mode != MODE_IMU_INVERTED ? 1 : -1));
```

Comment:
I think we can put the true part of the if statement it it's own block, per our convention.

But, are ternary operators in general "too tricky"?